### DISCUSSION TITLE
Github Copilot suggestions showing but not completing

### DISCUSSION THREAD

**Discussion by Verox001** [community/community#128147] [Copilot Conversations]
### Select Topic Area

Bug

### Body

**What steps will reproduce the issue?**
1. Install Github Copilot on remote host or on the client (for example WSL in my case) PHPStorm (or any other IDE)
2. Type a comment and wait for Copilot completion to show
3. Use Tab to apply the suggested completion, which is not working, since it only inserts a regular TAB

Jetbrains answered me here and said I should contact you, since this seems to be a problem with your plugin:
https://youtrack.jetbrains.com/issue/GTW-3086

That's the message I'm referring to: https://youtrack.jetbrains.com/issue/GTW-3086/Possible-remote-specific-error-Github-Copilot-suggestions-showing-but-not-completing#focus=Comments-27-9909535.0-0

I'm using PHPStorm 2024.1.3 with Github Copilot 1.5.6.5692 being installed on the Client.

**Comment by sanchezfranco** [comment]
I'm experiencing the same issue with PhpStorm. I've tried different versions of the IDE (2024.1.4 RC and 1.2), but the problem persists. I also attempted to change the hotkeys, but that didn't resolve it either.

**Comment by AttraktiveMedias** [comment]
Experiencing the same issue with PhpStorm too... PhpStorm 2024.3.5
